kuma/core/whitenoise.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangoWhiteNoise):
+
+    # Custom MIME types
+    KUMA_MIMETYPES = (
+        ('application/font-woff', '.woff'),)
+
+    # Use our custom MIME types, together with the default MIME types, in
+    # WhiteNoise. When our MIME types conflict with the defaults, use ours.
+    EXTRA_MIMETYPES = tuple({e[1]: e for e in reversed(
+        KUMA_MIMETYPES + DjangoWhiteNoise.EXTRA_MIMETYPES
+    )}.values())
```

Comment:
This class attribute will only be evaluated once, but it's kind of smelly code and to be honest I think we should not accidentally change the mimetypes we support when we update whitenoise and it adds a new entry. In other words, let's make this a static list of mimetypes we know we want to support and ignore the upstream mimetypes.